plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/GlueIcebergTableOperations.java

```diff
@@ -192,7 +193,10 @@ private void commitTableUpdate(Table table, TableMetadata metadata, BiFunction<T
     {
         String newMetadataLocation = writeNewMetadata(metadata, version.orElseThrow() + 1);
         TableInput tableInput = tableUpdateFunction.apply(table, newMetadataLocation);
-
+        if (MigrateProcedure.PROVIDER_PROPERTY_VALUE.equals(metadata.properties().get(MigrateProcedure.PROVIDER_PROPERTY_KEY))) {
+            // Assume this is a table executing migrate procedure
```

Comment:
Don't we reach this line from https://github.com/trinodb/trino/blob/eba2248bd30417d1440facb2a8067450ce669775/plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/AbstractIcebergTableOperations.java#L177 when we execute write operations after the migration is complete?